app/emails/_templates/public-cloud/TeamRequestCancellation.tsx

```diff
@@ -22,7 +23,7 @@ export default function TeamRequestCancellation({ request, requester }: EmailPro
       </Text>
 
       <LinkButton href={`/public-cloud/requests/${request.id}/request`}>View Request</LinkButton>
-
+      <Comment decisionComment={request.decisionComment} />
```

Comment:
the `Comment` email component should say whether it is a `Decision Comment` or `Cancellation Comment`